modules/packages/HDF5.chpl

```diff
@@ -3398,6 +3403,19 @@ module HDF5 {
     }
   }
   */
+  // Verify that the HDF5 version in use is at least as high as the
+  // (major, minor, release) arguments.
+  private proc verifyMinimumHDF5Version(major, minor, release) {
+    const HDF5Version = (C_HDF5.H5_VERS_MAJOR,
+                         C_HDF5.H5_VERS_MINOR,
+                         C_HDF5.H5_VERS_RELEASE),
+          requiredVersion = (major, minor, release);
+
+    if HDF5Version < requiredVersion {
+      halt("HDF5 version ", HDF5Version,
```

Comment:
I don't know if there's a good way to propagate the error out of the library since it happens during module initialization vs. in a regular function call.  We could make it an error that gets caught at the call to `verifyMinimumHDF5Version`, but then it's just a halt one level higher in the callstack.

Reply:
I see.. That's an interesting case for error-handling. Anyway, `halt` is fine for now. Thanks for explaining.